kotsadm/web/src/components/LicenseUploadProgress.jsx

```diff
@@ -34,13 +35,15 @@ class LicenseUploadProgress extends React.Component {
       });
 
       const response = await res.json();
-      
```

Comment:
we should reset the error message when this query succeeds, like `this.props.onError("");`